plugins/afr/validate.go

```diff
@@ -9,12 +9,28 @@ import (
 
 var names = [...]string{"replicate", "afr"}
 
+var shdEnabledKey = [...]string{"afr.self-heal-daemon", "replicate.self-heal-daemon"}
```

Comment:
`shdKeys` would a better name. The key name itself doesn't enable self heal, the value is what is taken into consideration.

Reply:
Wait, is `isVolReplicate` even used in `afr` package ? If not, please remove it.

Reply:
isVolReplicate is used in validateOptions() function in afr package